utils/local-deployment/gatewayapi-manifests/gwapi-gateway.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: Gateway
+metadata:
+  name: kuadrant-gwapi-gateway
+  namespace: kuadrant-system
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: default
+    hostname: "*.com"
```

Comment:
Why not unspecified?  According to https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.Listener
`When unspecified, all hostnames are matched.`

Reply:
Oh, I missed it! Thanks!